src/datatable.js

```diff
@@ -137,18 +138,26 @@ export class DataTable {
 
   checkDisabled(action, row) {
     if (typeof action.disabled === 'function') {
+      this.hasVisibleActions = true;
+
       return action.disabled(row);
     }
 
     return false;
   }
 
   checkVisibility(action, row) {
-    if (typeof action.visible === 'function') {
-      return action.visible(row);
+    if (typeof action.visible !== 'function') {
+      return true;
```

Comment:
that should set hasVisibleActions true, or? (in which case a =?: might be easier)